### PR TITLE
Fixes Chili sin carne

### PR DESCRIPTION
**Pull request by QuacksQ**
## About The Pull Request

I added a missing crafting recipe for Chili sin carne so that this can be crafted (Getting this recipe to be craft-able will also make other items possible to craft again since they use Chili sin carne as an ingredient)

## How This Contributes To The Skyrat Roleplay Experience

It gives us more foods to eat!

## Changelog

:cl:
Fix: fixes Chili sin carne being un-craftable
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm [hunk 1]
```diff
@@ -484,3 +484,15 @@
 	)
 	result = /obj/item/food/soup/red_porridge
 	subcategory = CAT_MOTH
+
+/datum/crafting_recipe/food/chili_sin_carne
```
Comment (Zonespace27) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]: make this modular please.

Comment (QuacksQ) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]: How would I do this?
(This is my first time doing anything with byond coding and github so im not really sure what you mean)

Comment (Zonespace27) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]: put the recipe datum in `modular_skyrat/modules/customization/datums/components/crafting/recipes.dm`

Comment (QuacksQ) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]: So the way I made it now right?

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm [hunk 2]
```diff
@@ -484,3 +484,4 @@
 	)
 	result = /obj/item/food/soup/red_porridge
 	subcategory = CAT_MOTH
+
```
Comment (Zonespace27) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]:
```suggestion
```

Comment (QuacksQ) [reply code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm]: Done 👍

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Thing I failed to notice earlier is that chili sin carne is a /tg/station recipe, so I'd recommend making the PR upstream on TG's repo instead of here.

**Comment by QuacksQ** [comment]
Upstream PR time

**Pull request closed without merging**